GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -860,17 +865,5 @@ private void InitFonts()
 
         private static Color getGrayTextColor(float degreeOfGrayness = 1f) =>
             ColorHelper.GetGrayTextColor(textColorName: KnownColor.ControlText, degreeOfGrayness);
-
-        internal TestAccessor GetTestAccessor() => new(this);
-
-        internal readonly struct TestAccessor
```

Comment:
@RussKie As IsDataLoadComplete was bypassing TestAccessor even if it is test-only, I assume you no longer requests that this property should be a TestAccessor as you requested in a previous review.